tox.ini

```diff
@@ -165,7 +165,8 @@ commands = {toxinidir}/scripts/generate_api_documentation.py --check-clean
 skipsdist = True
 skip_install = True
 deps =
-commands = {toxinidir}/scripts/check_doc_links.py
+    requests==2.27.1
+commands = python {toxinidir}/scripts/check_doc_links.py
```

Comment:
You need to install requests and then call python on the script;
![image](https://user-images.githubusercontent.com/35799987/173902932-48ab481c-7d0e-402a-be7b-4c328172a0fd.png)